### PR TITLE
docs(system-e2e): Add documentation on system-e2e

### PR DESCRIPTION
**Pull request by kristofer-lp8f**
## What
Add documentation on `system-e2e`

## Why
Docs are good :heart: 

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: handbook/repository/system-e2e.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+However, not all projects support this, or are incomplete in this setup. If this fails, find its `README.md` and follow the instructions given there. If that fails, reach out to the QA team and we’ll remedy the documentation and improve the initial setup.
+
+## 🤖 Start Playwright
+
+Playwright can be started in several ways. Using utility scripts, in container, on remote (dev), etc.
+
+<aside>
+💡 Add `TEST_ENVIRONMENT=dev` to the bottom of your `.env.secret` file to test against the live [dev-web](https://beta.dev01.devland.is/). Valid values are `local` (default), `dev`, `staging`, and `prod` to test the respective live environment.
+
+</aside>
```
Comment (kristofer-lp8f) [inline handbook/repository/system-e2e.md:31]:
```suggestion
{% hint style="info" %}
💡 Add `TEST_ENVIRONMENT=dev` to the bottom of your `.env.secret` file to test against the live [dev-web](https://beta.dev01.devland.is/). Valid values are `local` (default), `dev`, `staging`, and `prod` to test the respective live environment.
{% endhint %}
```

File: handbook/repository/system-e2e.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+- Utility script
+  - Get help: `./scripts/local-e2e.sh --help`
+  - Container: `./scripts/local-e2e.sh run -t acceptance -c container -i <app>`
+  - Source: `./scripts/local-e2e.sh run -t acceptance -c container -i <app>`
+  - ???: `./scripts/local-e2e.sh run -t acceptance -c container -i <app>`
+- Source code in dev-mode
+  - `yarn playwright install`
+  - `yarn playwright test --project <path/to/your/app>`
+
+You can append to the `test` command the path/name of your test case `yarn playwright test <path/to/spec/file>` to only test a specific spec.
```
[the remark this answers is not in the document]
Comment (kristofer-lp8f) [reply handbook/repository/system-e2e.md:42]:
```suggestion
{% hint style="info" %}
You can append to the `test` command the path/name of your test case `yarn playwright test <path/to/spec/file>` to only test a specific spec.
{% endhint %}
```

File: handbook/repository/system-e2e.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (saevarma) [reply handbook/repository/system-e2e.md]: Yeah remove it for now, but I find the 💡 more fund icon for info than the plain i, but that is subject to change in how gitbook renders the info hint.

File: package.json [hunk 3]
```diff
@@ -405,6 +406,9 @@
     "eslint-plugin-react": "7.29.4",
     "eslint-plugin-react-hooks": "4.4.0",
     "fetch-mock": "9.10.7",
+    "gitbook-plugin-hints": "^1.0.2",
+    "honkit": "^4.0.4",
```
Comment (saevarma) [inline package.json:410]: Why are you adding honkit packages? Our gitbook documentation is hosted on gitbook.com.

File: package.json [hunk 4]
```diff
@@ -405,6 +406,9 @@
     "eslint-plugin-react": "7.29.4",
     "eslint-plugin-react-hooks": "4.4.0",
     "fetch-mock": "9.10.7",
+    "gitbook-plugin-hints": "^1.0.2",
```
Comment (saevarma) [inline package.json:409]: The plugins work out of the box on gitbook.

**Review by saevarma — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (kristofer-lp8f) [reply package.json]: Tried HonKit, didn't manage a full migration, forgot to revert all

**Pull request merged**